resources/lib/syncEpisodes.py

```diff
@@ -578,10 +575,10 @@ def __compareShows(self, shows_col1, shows_col2, rating=False, restrict=False):
 
                 if show_col2:
                     show = {'title': show_col1['title'], 'ids': {}, 'year': show_col1['year']}
-                    if 'tvdb' in show_col1['ids']:
-                        show['ids'] = {'tvdb': show_col1['ids']['tvdb']}
-                    if 'imdb' in show_col2 and show_col2['imdb']:
-                        show['ids']['imdb'] = show_col2['imdb']
+                    if show_col1['ids']:
+                        show['ids'].update(show_col1['ids'])
+                    if show_col2['ids']:
+                        show['ids'].update(show_col2['ids'])
```

Comment:
Ok, lets see, I might be misunderstanding something.

My take is that we already matched col1 and col2, so they should refer to the same show and any id they both have, it should be the same. Is it possible that we have the same show with different ids in trakt?

There is a risk that the match happened by tittle which may be wrong. I thought this risk was assumed. One idea to alleviate the risk is to only attempt match by title if the local info has no id at all.  But if there is any id in the local show info and we cant match the id, the assume that there is no match instead of attempting match by title. But this would be an improvement for a different patch.